jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -166,6 +162,28 @@ private final Path checkAliasPath()
         }
         return null;
     }
+    
+    private boolean isSame(Path pathA, Path pathB)
+    {
+        int aCount = pathA.getNameCount();
+        int bCount = pathB.getNameCount();
+        if (aCount != bCount)
+        {
+            // different number of segments
+            return false;
+        }
+        
+        // compare each segment of path, backwards
+        for (int i = bCount - 1; i >= 0; i--)
```

Comment:
I prefer to write this loop as : 
```java
for (int i = bCount; i-- > 0; )
```
Does work so well with the checkstyle required spacing, but I like the arrow notation :)